nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -28,6 +28,31 @@ The following advisory is available for the {run-once-operator} 1.1.2:
 
 * This release of the {run-once-operator} addresses several Common Vulnerabilities and Exposures (CVEs).
 
+[id="run-once-duration-override-operator-release-notes-1-1-1"]
+== Run Once Duration Override Operator 1.1.1
+
+Issued: 1 July 2024
+
+The following advisory is available for the {run-once-operator} 1.1.1: link:https://access.redhat.com/errata/RHSA-2024:1616[RHSA-2024:1616]
+
+[id="RODOO-operator-1-1-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can install and use the {run-once-operator} in an {product-title} cluster running in FIPS mode.
++
+--
+[IMPORTANT]
+====
+To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base-full} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
+When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
```

Comment:
```suggestion
When you run {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
```
Tiny rewrite to avoid potential dangling modifier: https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#grammar